runtime/doc/usr_05.txt

```diff
@@ -245,7 +246,45 @@ The ":map" command (with no arguments) lists your current mappings.  At
 least the ones for Normal mode.  More about mappings in section |40.1|.
 
 ==============================================================================
-*05.4*	Adding a plugin					*add-plugin* *plugin*
+*05.4*	Adding a package			*add-package* *matchit-install*
+
+A package is a set of files that you can add to Vim.  There are two kinds of
+packages: optional and automatically loaded on startup.
+
+The Vim distribution comes with a few packages that you can optionally use.
+For example, the vimball plugin.  This plugin supports creating and using
+vimballs (self-installing Vim plugin archives).
+
+To start using the vimball plugin, add one line to your vimrc file: >
+	packadd vimball
+
+That's all!  You can also type the command to try it out.  Now you can find
+help about this plugin: >
+	:help vimball
```

Comment:
Yes, but very few users who actually need this documentation, would pick up on that bit of domain knowledge.

Though I suppose most of this will continue to be abstracted by package managers.